Source/WebCore/rendering/RenderImage.cpp

```diff
@@ -811,10 +807,10 @@ void RenderImage::updateAltText()
     if (!element())
         return;
 
-    if (is<HTMLInputElement>(*element()))
-        m_altText = downcast<HTMLInputElement>(*element()).altText();
-    else if (is<HTMLImageElement>(*element()))
-        m_altText = downcast<HTMLImageElement>(*element()).altText();
+    if (auto input = dynamicDowncast<HTMLInputElement>(*element()))
+        m_altText = input->altText();
+    else if (auto image = dynamicDowncast<HTMLImageElement>(*element()))
+        m_altText = image->altText();
```

Comment:
RefPtr?

Reply:
Just for getting the altText(). It may not be super critical. But I do think we should either use `auto*` here to `RefPtr` rather than `auto`.